README.md

```diff
@@ -185,7 +185,7 @@ This website is available as open source under the terms of the
 [docker-install]: https://www.docker.com/
 [docker]: #docker
 [gems]: https://rubygems.org/
-[gh-actions-badge]: https://github.com/SwedbankPay/developer.swedbankpay.com/workflows/jekyll-build/badge.svg
+[gh-actions-badge]: https://github.com/SwedbankPay/developer.swedbankpay.com/workflows/Test/badge.svg
```

Comment:
```suggestion
[gh-actions-badge]: https://github.com/SwedbankPay/developer.swedbankpay.com/workflows/test/badge.svg
```
Change `Test` to `test` in URL for consistencty. 😉

Reply:
The name of the test is `Test`, so that unfortunately doesn't work.